src/js/components/Select/stories/Disabled.js

```diff
@@ -1,26 +1,24 @@
 import React, { useState } from 'react';
-
-import { Box, Grommet, Select, Text } from 'grommet';
-import { grommet } from 'grommet/themes';
+import { Box, Select, Text } from 'grommet';
 
 export const Disabled = () => {
   const options = ['one', 'two'];
   const [value, setValue] = useState('');
   return (
-    <Grommet full theme={grommet}>
-      <Box fill align="center" justify="start" pad="large" gap="medium">
-        <Text weight="bold">Disabled</Text>
-        <Select
-          id="select"
-          name="select"
-          placeholder="Select"
-          value={value}
-          options={options}
-          onChange={({ option }) => setValue(option)}
-          disabled
-        />
-      </Box>
-    </Grommet>
+    // Uncomment <Grommet> lines when using outside of storybook
+    // <Grommet theme={...}>
+    <Box fill align="center" justify="start" pad="large" gap="medium">
+      <Text weight="bold">Disabled</Text>
+      <Select
+        id="select"
+        name="select"
+        placeholder="Select"
+        value={value}
+        options={options}
+        onChange={({ option }) => setValue(option)}
+        disabled
+      />
+    </Box>
```

Comment:
```suggestion
    </Box>
    // </Grommet>
```
We should include the Grommet closing tag in a comment